Makefile

```diff
@@ -71,9 +71,14 @@ format: install-goimports
 	@echo "> Formating code..."
 	test -z $$(goimports -l -w -local github.com/buildpacks/lifecycle $$(find . -type f -name '*.go' -not -path "*/vendor/*"))
 
+verify-jq:
+	ifeq (, $(shell which jq))
```

Comment:
I played around with this and I think Travis is failing because of the indentation here. `ifeq` should have the same indentation as the target.

Reply:
yeah, I figured that out eventually. pushed changes!